combat-trainer.lic

```diff
@@ -842,6 +842,18 @@ class SpellProcess
 
     @perc_health_timer = Time.now
 
+    @check_discern_settings = OpenStruct.new(
```

Comment:
You _probably_ need to `require 'ostruct'` at the top of the file if you're going to do this. Have you tested this using a Hash instead of OpenStruct?

Reply:
I used a hash initially but switched to openstruct due to errors from DRCA.  Common-arcana::check_discern takes user settings which is an OpenStruct. 

I can add the require to the top. I think the reason I did not see any errors is that dependency does this.

Reply:
The alternative I considered was making settings an instance variable, but none of the other Combat processes seem to do this.

Reply:
using the dot operator on hashes was the error if memory serves
```
2.0.0-p647 :002 > settings_hash = {'a'=> 'b'}
 => {"a"=>"b"}
2.0.0-p647 :003 > require 'ostruct'
 => true
2.0.0-p647 :004 > settings_struct = OpenStruct.new('a' => 'b')
 => #<OpenStruct a="b">
2.0.0-p647 :005 > settings_hash.a
NoMethodError: undefined method `a' for {"a"=>"b"}:Hash
        from (irb):5
        from /home/jonas/.rvm/rubies/ruby-2.0.0-p647/bin/irb:12:in `<main>'
```

Reply:
Storing `settings` seems pretty reasonable to me

Reply:
OK, that is what I would like to do as well. Is it alright if I store settings in an instance variable in this PR and then refactor the SpellProcess to use the settings instance var in another PR?